backend/exchanges/btcdirect.go

```diff
@@ -98,7 +100,24 @@ func BtcDirectDeals() *ExchangeDealsList {
 				Payment:  BancontactPayment,
 				IsHidden: true,
 			},
-		},
+		}
+	case SellAction:
+		deals = []*ExchangeDeal{
+			{
+				Fee:     4.9, // 4.9%,
+				Payment: CardPayment,
+				IsFast:  true,
```

Comment:
CardPayment  option can be removed here. (was already wrong before in the hardcoded option)

Reply:
thank you